app/models/bulkrax/csv_collection_entry.rb

```diff
@@ -7,13 +7,27 @@ def factory_class
     end
 
     def build_metadata
-      self.parsed_metadata = self.raw_metadata
+      self.parsed_metadata = {}
+      self.parsed_metadata[work_identifier] = self.identifier
```

Comment:
where does identifier come from?

Reply:
[CsvParser#unique_collection_identifier](https://github.com/samvera-labs/bulkrax/pull/364/files#diff-ccbdc46eb1fafd2a6aec08c257c2589531c46fee5d793832a85bf0653f754490R275-R284). I added a comment to the top of the new method that handles this explaining why we're using `self.identifier`